sysutils/pfSense-pkg-LCDproc/files/usr/local/www/packages/lcdproc/lcdproc_screens.php

```diff
@@ -253,6 +255,16 @@
 	)
 );
 
+$section->addInput(
+	new Form_Checkbox(
+		'scr_ntp', // checkbox name (id)
+		'NTP Status', // checkbox label
+		'Display NTP status', // checkbox text
+		$pconfig['scr_ntp'] // checkbox initial value
+	)
+);
+//)->setHelp('Display NTP status information');
```

Comment:
Why not keep this help string? It would be useful. Also worth noting that it would only work if the NTP service is enabled/active.

If it's not needed, remove the line.

Reply:
removed